src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaAndFullInfo {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    abstract void fullInfo();
```

Comment:
1. create interface for that, like AreaCalculator
2. dont invent your own name, use `draw` as method name and `Drawable` as interface name
```suggestion
```